feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -46,10 +46,6 @@ import kotlinx.coroutines.launch
 private const val TAG = "PreviewViewModel"
 private const val IMAGE_CAPTURE_TRACE = "JCA Image Capture"
 
-// toast test descriptions
-const val IMAGE_CAPTURE_SUCCESS_TOAST_TAG = "ImageCaptureSuccessToast"
```

Comment:
I didn't find a way to assign tag to a snackBar because in compose it is launched differently than a normal compose component. We actually couldn't directly customize the snackBar. In the test, I changed the places that looks for this tag into looking for the text of the snackBar, which worked. Since this tag is then unused, I just deleted it.

Reply:
To keep things simplified, I didn't directly create a SnackBar component. Instead, I used snackbarHostState.showSnackbar, which allows the user to show a snackBar by passing in attributes or SnackBarVisuals without having created a SnackBar component. Unfortunately, this method doesn't allow testTags to be assigned.

The tests are passing because I changed the places that look for this tag to looking for the SnackBar text. This tag ended up being unused so I just deleted it here

Reply:
Not being able to use tags was an issue with system toasts too...

Maybe we can reuse the "testable" part of testable toast. Triggering an invisible composable to render in the background which can be assigned a tag and discoverable by the integration tests.

Also by decoupling that "testable" part from the testable toasts, we can add a condition so that the invisible component is only rendered on testing builds. 

I think for now this is ok, but i can quickly make a PR in the near future to make that testable component

Reply:
Done. Display an invisible spacer with testTag if the snackBar is shown. Though for some reason only uiAutomator can detect this but composeTestRule can't. I had to change set_screen_flash_and_capture_with_screen_change_overlay_shown() in FlashDeviceTest to use UiAutomator